src/Compilers/VisualBasic/Portable/Symbols/AssemblySymbol.vb

```diff
@@ -576,48 +576,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Friend MustOverride ReadOnly Property PublicKey As ImmutableArray(Of Byte)
 
-        Protected Enum IVTConclusion
-            Match
-            OneSignedOneNot
-            PublicKeyDoesntMatch
-            NoRelationshipClaimed
-        End Enum
-
-        Protected Function PerformIVTCheck(key As ImmutableArray(Of Byte), otherIdentity As AssemblyIdentity) As IVTConclusion
-            ' Implementation of this function in C# compiler is somewhat different, but we believe
-            ' that the difference doesn't affect any real world scenarios that we know/care about.
-            ' At the moment we don't feel it is worth porting the logic, but we might reconsider in the future. 
```

Comment:
That moment has passed. But rather than porting the logic we moved it to a shared layer.